packages/dashboard-frontend/src/store/Workspaces/devWorkspaces/index.ts

```diff
@@ -365,6 +365,9 @@ export const actionCreators: ActionCreators = {
 
         getDevWorkspaceClient().checkForDevWorkspaceError(startingWorkspace);
       } catch (e) {
+        if (common.helpers.errors.includesAxiosResponse(e) && isOAuthResponse(e.response.data)) {
+          return;
+        }
```

Comment:
@vinokurig could you please add a comment in the code why we skip exception in this case?

Reply:
done